guides/v15-mdc-migration.md

```diff
@@ -0,0 +1,571 @@
+# Migrating to MDC-based Angular Material Components
+
+In Angular Material v15, many of the components have been rewritten to be based on the official 
+[Material Design Components for Web (MDC)](https://github.com/material-components/material-components-web).
+The components in the following packages have been rewritten:
+* @angular/material/autocomplete
+* @angular/material/button
+* @angular/material/card
+* @angular/material/checkbox
+* @angular/material/chips
+* @angular/material/core
+* @angular/material/dialog
+* @angular/material/form-field
+* @angular/material/input
+* @angular/material/list
+* @angular/material/menu
+* @angular/material/paginator
+* @angular/material/progress-bar
+* @angular/material/progress-spinner
+* @angular/material/radio
+* @angular/material/select
+* @angular/material/slide-toggle
+* @angular/material/slider
+* @angular/material/snack-bar
+* @angular/material/table
+* @angular/material/tabs
+* @angular/material/tooltip
+
+The rewritten components offer several benefits over the old implementations, including:
+* Improved accessibility
+* Better adherence to the Material Design spec
+* Faster adoption of future versions of the Material Design spec, due to being based on common 
+  infrastructure
+
+## What has changed?
+
+The new components have different internal DOM and CSS styles. However, the TypeScript APIs and
+component/directive selectors for the new components have been kept as close as possible to the old
+implementation. This makes it straightforward to migrate your app and get it running with the new
+components.
+
+Due to the new DOM and CSS, you will likely find that some styles in your app need to be adjusted,
+particularly if your CSS is overriding styles on internal elements on any of the migrated
+components.
+
+There are a few components with larger changes to their APIs that were necessary in order to
+integrate with MDC. These components include:
+* form-field
+* chips
+* slider
+* list
+
+See below for a [detailed list of changes](#Detailed List of Changes) to be aware of for these and
+other components.
+
+The old implementation of each new component is now deprecated, but still available under a "legacy"
+package. e.g. The old `mat-button` implementation can be used by importing the legacy button module.
+
+```ts
+import {MatLegacyButtonModule} from '@angular/material/legacy-button';
+```
+
+## How to Migrate
+
+Migrating an existing app to use MDC based components is a tool-assisted migration. The Angular
+Material team has built [schematics](https://angular.io/guide/schematics) to handle as much of the
+work as possible. However, some manual effort will be needed.
```

Comment:
```suggestion
You can start your migration by running Angular Material's automated refactoring
tool. This tool, implemented as an [Angular Schematic](https://angular.io/guide/schematics),
updates the majority your code to the new component versions. While some follow-up is
necessary, you can reduce the manual effort by following the following best practices:
```

(this was a hard one) rough suggestion: some passive voice, some tense (trying to always keep present tense)

Reply:
sgtm, just changed the part that says "following the following" because that sounded silly